api/routes/utils.js

```diff
@@ -331,15 +331,15 @@ var utils = {
     return true;
   },
 
-  // Sort users by ROI for a given category, increasing order
-  getROIComparator: function(category) {
+  // Sort users by dividends for a given category, increasing order
+  getDividendsComparator: function(category) {
     return function(a, b) {
       var indexA = this.getPortfolioIndex(a, category);
       var indexB = this.getPortfolioIndex(b, category);
 
-      var roiA = a.portfolio[indexA].roi.value;
-      var roiB = b.portfolio[indexB].roi.value;
-      return roiA - roiB;
+      var dividendsA = a.portfolio[indexA].dividends.value;
```

Comment:
I think this will need to be changed a little bit, since each dividend is conceptually tied to an investment. See above comment for handling this with another aggregated field at the portfolio category level or a utils function